packages/muelu/research/regionMG/src/SetupRegionHierarchy_def.hpp

```diff
@@ -1042,73 +1009,73 @@ void createRegionHierarchy(const int maxRegPerProc,
+//                           Array<RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > >& compRowMaps,
+//                           Array<RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > >& compColMaps,
+//                           Array<std::vector<RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > > >& regRowMaps,
+//                           Array<std::vector<RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > > >& regColMaps,
+//                           Array<std::vector<RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > > >& quasiRegRowMaps,
+//                           Array<std::vector<RCP<Xpetra::Map<LocalOrdinal, GlobalOrdinal, Node> > > >& quasiRegColMaps,
+//                           Array<std::vector<RCP<Xpetra::Matrix<Scalar, LocalOrdinal, GlobalOrdinal, Node> > > >& regMatrices,
+//                           Array<std::vector<RCP<Xpetra::Matrix<Scalar, LocalOrdinal, GlobalOrdinal, Node> > > >& regProlong,
+//                           Array<std::vector<RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> > > >& regRowImporters,
+//                           Array<Array<RCP<Xpetra::Vector<Scalar, LocalOrdinal, GlobalOrdinal, Node> > > >& regInterfaceScalings,
+//                           Array<RCP<Teuchos::ParameterList> >& smootherParams
+//                           )
+//{
+//  // Define dummy values
+//  const int maxRegPerGID = 0;
+//  ArrayView<LocalOrdinal> compositeToRegionLIDs = {};
+//  RCP<ParameterList> coarseSolverParams = rcp(new ParameterList("Coarse solver parameters"));
+//  coarseSolverParams->set<bool>("use coarse solver", true);
+//  RCP<ParameterList> dummy = Teuchos::parameterList();
+//  Array<RCP<Xpetra::MultiVector<GlobalOrdinal, LocalOrdinal, GlobalOrdinal, Node> > > interfaceGIDsPerLevel;
+//  Array<RCP<Xpetra::MultiVector<LocalOrdinal, LocalOrdinal, GlobalOrdinal, Node> > > regionsPerGIDWithGhostsPerLevel;
+//  Array<ArrayRCP<LocalOrdinal> > regionMatVecLIDsPerLevel;
+//  Array<RCP<Xpetra::Import<LocalOrdinal, GlobalOrdinal, Node> > > regionInterfaceImporterPerLevel;
+//
+//  // Call the actual routine
+//  createRegionHierarchy(maxRegPerProc, numDimensions, lNodesPerDim,
+//                        aggregationRegionType, dummy, xmlFileName, nullspace, coordinates,
+//                        regionGrpMats, mapComp, rowMapPerGrp, colMapPerGrp, revisedRowMapPerGrp,
+//                        revisedColMapPerGrp, rowImportPerGrp, compRowMaps, compColMaps, regRowMaps,
+//                        regColMaps, quasiRegRowMaps, quasiRegColMaps, regMatrices, regProlong,
+//                        regRowImporters, regInterfaceScalings, interfaceGIDsPerLevel,
+//                        regionsPerGIDWithGhostsPerLevel, regionMatVecLIDsPerLevel,
+//                        regionInterfaceImporterPerLevel, maxRegPerGID,
+//                        compositeToRegionLIDs, coarseSolverParams, smootherParams, dummy, false);
+//}
```

Comment:
I think you could/should delete this commented code, as it's not gonna ever be reactivated.